client/package.json

```diff
@@ -15,7 +15,6 @@
     "@types/eslint": "^8.4.6",
     "expo": "~46.0.9",
     "expo-status-bar": "~1.4.0",
-    "mongoose": "^6.5.4",
```

Comment:
Is this supposed to be removed?

Reply:
It doesn't need to be in the client folder. It should be in the server's package.json.